### PR TITLE
fix: add default inbound

### PR DESCRIPTION
**Pull request by iamar7**
### Description
Set inbound as the default direction.

https://github.com/terraform-ibm-modules/terraform-ibm-security-group/issues/219

### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [x] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

##### Release notes content
This release sets the default direction of security group rule as `inbound` in the module. 

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

**Comment by ocofaigh** [comment]
Why would we open that up as a default value and make it unsecure? I think the request is to make `direction` optional in the `security_group_rules` and default it to `inbound` if not specified

File: variables.tf [hunk 1]
```diff
@@ -77,7 +77,6 @@ variable "security_group_rules" {
       )
     })
   )
-  default = []
```
Comment (ocofaigh) [inline variables.tf:80]: put back in default value of empty list

Comment (iamar7) [reply variables.tf]: Done

**Review by ocofaigh — APPROVED** [review]

**Review by ocofaigh — APPROVED** [review]

**Pull request merged**